change/@fluentui-web-components-79615c7f-6948-4402-88f1-e8a020cc9aaf.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

```suggestion
  "type": "minor",
```
 since it contains api change?